org.omg.sysml.jupyter.kernel/src/main/java/org/omg/sysml/jupyter/kernel/magic/View.java

```diff
@@ -0,0 +1,68 @@
+/*
+ * SysML 2 Pilot Implementation
+ * Copyright (c) 2020 Mgnite Inc.
+ *
```

Comment:
Maybe you should add your copyright notice.

Reply:
Oops. I fixed it.